hack/test-e2e-local.sh

```diff
@@ -55,6 +55,12 @@ if [[ "$TYPE" == "operator" ]] || [[ "$TYPE" == "operator-seed" ]]; then
       exit 1
     fi
   fi
+
+  if [[ "$TYPE" == "operator-seed" ]]; then
+    # /etc/hosts must be updated before garden can be created
+    echo "> Deploying Garden and Soil"
+    make operator-seed-up
+  fi
```

Comment:
Is the comment still true? If so this has moved to a wrong position as it is now before the `/etc/hosts` manipulation.

Reply:
It's still true - a few lines above, the `/etc/hosts` file is manipulated with the garden-specific domain names.